example.py

```diff
@@ -0,0 +1,264 @@
+    request.tx_hash = tx_hash
+
+    channel = grpc.insecure_channel('127.0.0.1:50051')
+    stub = endpoint_pb2_grpc.CommandServiceStub(channel)
+
+    response = stub.Status(request)
+    status = endpoint_pb2.TxStatus.Name(response.tx_status)
+    print("Status of transaction is:", status)
+
+    if status != "COMMITTED":
+        print("Your transaction wasn't committed")
+        exit(1)
+
+
+def print_status_streaming(tx):
+    # Create status request
+
+    print("Hash of the transaction: ", tx.hash().hex())
+    tx_hash = tx.hash().blob()
+
+    # Check python version
+    if sys.version_info[0] == 2:
+        tx_hash = ''.join(map(chr, tx_hash))
+    else:
+        tx_hash = bytes(tx_hash)
+
+    # Create request
+    request = endpoint_pb2.TxStatusRequest()
+    request.tx_hash = tx_hash
+
+    # Create connection to Iroha
+    channel = grpc.insecure_channel('127.0.0.1:50051')
+    stub = endpoint_pb2_grpc.CommandServiceStub(channel)
+
+    # Send request
+    response = stub.StatusStream(request)
+
+    for status in response:
+        print("Status of transaction:")
+        print(status)
+
+def send_tx(tx, key_pair):
+    tx_blob = proto_tx_helper.signAndAddSignature(tx, key_pair).blob()
+    proto_tx = block_pb2.Transaction()
+
+    if sys.version_info[0] == 2:
+        tmp = ''.join(map(chr, tx_blob))
+    else:
+        tmp = bytes(tx_blob)
+
+    proto_tx.ParseFromString(tmp)
+
+    channel = grpc.insecure_channel('127.0.0.1:50051')
+    stub = endpoint_pb2_grpc.CommandServiceStub(channel)
+
+    stub.Torii(proto_tx)
+
+
+def send_query(query, key_pair):
+    query_blob = proto_query_helper.signAndAddSignature(query, key_pair).blob()
+
+    proto_query = queries_pb2.Query()
+
+    if sys.version_info[0] == 2:
+        tmp = ''.join(map(chr, query_blob))
+    else:
+        tmp = bytes(query_blob)
+
+    proto_query.ParseFromString(tmp)
+
+    channel = grpc.insecure_channel('127.0.0.1:50051')
+    query_stub = endpoint_pb2_grpc.QueryServiceStub(channel)
+    query_response = query_stub.Find(proto_query)
+
+    return query_response
+
+
+########################
+class Account:
+    def __init__(self, name, keypair):
+        self.name = name
+        self.keypair = keypair
+
+def add_user(user):
+    global tx_counter
+    tx = tx_builder.creatorAccountId(creator) \
+        .txCounter(tx_counter) \
+        .createdTime(get_time()) \
+        .createAccount(user.name, 'global', user.keypair.publicKey()) \
+        .appendRole(user.name + '@global', 'voter') \
+        .build()
+
+    tx_counter += 1
+
+    send_tx(tx, key_pair)
+    print_status_streaming(tx)
+
+def add_candidate(candidate, domain, tx_buidler):
+    return tx_buidler \
+        .createAccount(candidate.name, domain, candidate.keypair.publicKey()) \
+        .appendRole(candidate.name + '@' + domain, 'candidate')
+
+def add_voter(voter, domain, tx_buidler):
+    return tx_buidler \
+        .appendRole(voter.name + '@global', 'prevote') \
```

Comment:
why?

Reply:
to add asset (vote) for account and then prohibit transfer inter accounts.